cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/workflow/DefaultWorkflowConditionConfigurer.java

```diff
@@ -76,8 +78,8 @@ public DefaultWorkflowConditionConfigurer(T parentConfigurer, String predicateCl
   @SuppressWarnings("unchecked")
   public WorkflowConditionConfigurer<? extends WorkflowConditionConfigurer<T>> condition(
     Predicate<WorkflowContext> predicate) {
-    String predicateClassName = predicate.getClass().getName();
-    return new DefaultWorkflowConditionConfigurer<>(this, predicateClassName);
+    return new DefaultWorkflowConditionConfigurer<>(predicate.getClass().getSimpleName(), this,
```

Comment:
can we just pass ``predicate`` to ``DefaultWorkflowConfigurer`` instead of passing both ``predicate.getClass().getSimpleName()`` and ``predicate.getClass().getName()``